### PR TITLE
hotel app with database

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: flask_orm/rooms/routes.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        if value:
+            return RoomsModel.query.get(value)
+        return filter_items_by_args(args, RoomsModel, RoomsModel.number)
+
+    def post(self):
+        data = json.loads(request.data)
+        new_room = RoomsModel(**data)
+        db.session.add(new_room)
+        db.session.commit()
+        return "Successfully added a new room"
```
Comment (dimigor) [inline flask_orm/rooms/routes.py]: return created room as response

File: flask_orm/rooms/routes.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    @marshal_with(rooms_structure)
+    def put(self, value):
+        data = json.loads(request.data)
+        room = RoomsModel.query.get(value)
+        for key, value in data.items():
+            setattr(room, key, value)
+            room.status = "closed" if key == "tenant_id" else "available"
+        db.session.commit()
+        return "Successfully updated a room"
```
Comment (dimigor) [inline flask_orm/rooms/routes.py]: return updated room as a response

File: flask_orm/staffs/__init__.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from flask import Blueprint
+from flask_restful import Api
+from .routes import Staffs
+
+staffs = Blueprint("staffs", __name__)
+api_rooms = Api(staffs)
+
+api_rooms.add_resource(Staffs, "/staffs", "/staffs/<value>")
```
Comment (dimigor) [inline flask_orm/staffs/__init__.py]: endline
do not miss them

File: flask_orm/rooms/routes.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    @marshal_with(rooms_structure)
+    def get(self, value=None):
+        args = room_parser.parse_args()
+        if value:
+            return RoomsModel.query.get(value)
+        return filter_items_by_args(args, RoomsModel, RoomsModel.number)
+
+    def post(self):
+        data = json.loads(request.data)
+        new_room = RoomsModel(**data)
```
Comment (dimigor) [inline flask_orm/rooms/routes.py]: what about cases when something went wrong?

File: flask_orm/rooms/routes.py (hunk 1 above)
Comment (sudakevych-andrii) [reply flask_orm/rooms/routes.py]: Done

File: flask_orm/rooms/routes.py (hunk 2 above)
Comment (sudakevych-andrii) [reply flask_orm/rooms/routes.py]: Done

File: flask_orm/staffs/__init__.py (hunk 3 above)
Comment (sudakevych-andrii) [reply flask_orm/staffs/__init__.py]: Done

File: flask_orm/rooms/routes.py (hunk 4 above)
Comment (sudakevych-andrii) [reply flask_orm/rooms/routes.py]: Done

**Pull request merged**